scripts/DESeq_functions.R

```diff
@@ -101,7 +101,12 @@ get_DESeq_results <- function(dds, exp_metadata, contrasts, design, params, curr
 
         compte <- Counts[Filter[,1] == 1,]
         Filter <- Filter[rownames(Filter) %in% rownames(compte), , drop = F]
-
+        
+        #save all genes that are present regardless of counts 
+        dfGenesPassed <- data.frame(Ensembl_Gene_ID = rownames(subset(Filter, Filter[, 1] == 1)))
+        dfGenesFailed <- data.frame(Ensembl_Gene_ID = rownames(subset(Filter, Filter[, 1] == 0)))
+        dfGenes <- rbind(dfGenesPassed, dfGenesFailed)
```

Comment:
So, rownames(Counts) should be equivalent to rownames(Filter), as per above.

Additionally - subsetting the Filter table twice and recombining them isn't necessary since rownames(Filter) would be equivalent to that

Can you test replacing lines 106 to 108 with

dfGenes <- rownames(Counts) ?

Reply:
Thank you for the feedback, you are right this is redundant. I have incorporated your suggestion and checked md5sums to find they are equivalent.

Reply:
> Thank you for the feedback, you are right this is redundant. I have incorporated your suggestion and checked md5sums to find they are equivalent.

Thanks for checking this!